tests/robustness/failpoint/trigger.go

```diff
@@ -72,7 +72,7 @@ func (t triggerCompact) Trigger(ctx context.Context, _ *testing.T, member e2e.Et
 			return nil, fmt.Errorf("failed to get revision: %w", err)
 		}
 
-		if !t.multiBatchCompaction || rev > int64(clus.Cfg.ServerConfig.ExperimentalCompactionBatchLimit) {
+		if !t.multiBatchCompaction || rev >= int64(clus.Cfg.ServerConfig.ExperimentalCompactionBatchLimit) {
```

Comment:
How this change is related? Does it help with repro? I assumed ">" was intentional as we want have additional revision to ensure 2 batches execute.

Reply:
Yeah. Reverted this change.